src/features/cheatcodes/pages/AppComponents.tsx

```diff
@@ -473,9 +474,15 @@ export const AppComponents: FunctionComponent = () => {
         <AlignedText>
           <LocationBuilding size={24} />
           <Text> - LocationBuilding </Text>
-          <TicketBooked width={48} height={24} />
+        </AlignedText>
+        <AlignedText>
+          <TicketBooked width={24} height={24} />
```

Comment:
Utiliser `getSpacing` n'est plus d'actualité ? Si c'est le cas, il va falloir en discuter demain en point standard.
```suggestion
          <TicketBooked width={getSpacing(6)} height={getSpacing(6)} />
```

Reply:
Si si c'est toujours d'actualité ! Je modifie